packages/app-bridge/src/AppBridgePlatformApp.ts

```diff
@@ -94,6 +94,14 @@ export class AppBridgePlatformApp implements IAppBridgePlatformApp {
         'Context.connected': new Map(),
     };
 
+    constructor() {
+        if (window.appBridgePlatformApp) {
+            return window.appBridgePlatformApp;
```

Comment:
This is an ignorant question but I'm not knowledgable about platform apps at all. Is it possible for there to ever be two platform apps on the same page? The pr description mentions for each iframe I'm not sure what the iframe is. Also would there be any conflict with the migration to the SPA structure? For example, there may only ever be one instance of this class on the window for the whole time the user is navigating through the website, including changing pages. Is this intended?

Reply:
No for now we will not have more then 1 PlatformApp per iframe. That will not change anytime soon. The window will then also be the one of the iframe itself and not the global window therefore an SPA structure doesn't have changes for this one.